sdk/tables/data-tables/src/serialization.ts

```diff
@@ -72,10 +77,11 @@ function getSerializedValue(value: any): serializedType {
 export function serialize(obj: object): object {
   const serialized: any = {};
   for (const [key, value] of Object.entries(obj)) {
+    const transformedKey = Object.keys(propertyCaseMap).find(k => propertyCaseMap[k] === key) || key;
```

Comment:
If you're going to do the work of calling `Object.keys`, why not have the map be a native Map?

Then this could become

```ts
const transformedKey = propertyCaseMap.get(key) ?? key;
```

Reply:
so it would be

```typescript

const propertyMap = new  Map<string, string>([
  ["PartitionKey", "partitionKey"],
  ["RowKey", "rowKey"]
]);
```

Here I'd need to translate from `partitionKey` to to `PartitionKey` which is from value -> key

This would get a value from a key right?
```typescript
const transformedKey = propertyCaseMap.get(key) ?? key;
```

Reply:
That's correct, though you'll have to use the same map name in both snippets. ;)

Reply:
Is there a nice way to get a key from a value using the native Map?